itou/templates/prescribers/email/validated_prescriber_organization_email_body.txt

```diff
@@ -0,0 +1,16 @@
+{% extends "layout/base_email_text_body.txt" %}
+{% load format_filters %}
+{% load i18n %}
+{% block body %}
+
+{% trans "La vérification de l'habilitation de l'organisation que vous avez saisi lors de votre inscription est terminée." %}
+
+{% trans "*Structure* :" %}
+
+- {% trans "Nom" %} : {{ organization.display_name }}
+- {% trans "SIRET" %} : {{ organization.siret }}
+- {% trans "Type" %} : {{ organization.kind }}
+
+{% trans "Vous recevez cet email en tant que propriétaire d'un compte associé à l'organisation ci-dessus." %}
```

Comment:
Le mot "propriétaire" ne colle pas si tu envoies le mail à `to = [u.email for u in self.active_members]`.

> En tant que collaborateur au sein de cette organisation

De toute façon je pense que le métier changera les textes si tu demande un test de ta recette.

Reply:
Je laisse le cas ouvert, et notifie... En attendant, je mets collaborateur